server/store/ip_store.go

```diff
@@ -75,10 +75,10 @@ func RegisterIPStoreDriver(name string, driver IPStoreDriver) {
 }
 
 // OpenIPStore returns an IPStore specified by a configuration.
-func OpenIPStore(cfg *DriverConfig) (IPStore, error) {
+func OpenIPStore(cfg *DriverConfig) (IPStore, StopperFunc, error) {
```

Comment:
I'm not sure what the issue being discussed is anymore. Embedding an exported type being exposed is completely unrelated to how you can implement a `Stop` method on a type that calls the stop methods on it's contained members: https://play.golang.org/p/bd9bHfW8Gf